app/ts/ClaimFundsUI/ClaimFunds.tsx

```diff
@@ -6,9 +6,18 @@ import { getAvailableDisputes, getAvailableReports, getAvailableShareData, redee
 import { forkReportingParticipants, getAvailableDisputesFromForkedMarkets } from '../utils/augurExtraUtilities.js'
 import { ReadClient, WriteClient } from '../utils/ethereumWallet.js'
 
+const removeIfExistsAddOtherwise = (array: readonly AccountAddress[], newEntry: AccountAddress) => {
+	if (array.find((entry) => entry === newEntry)) {
+		return array.filter(((entry) => entry !== newEntry))
+	} else {
+		return [...array, newEntry]
+	}
+}
+
+const exists = (array: readonly AccountAddress[], newEntry: AccountAddress) => array.find((entry) => entry === newEntry) !== undefined
```

Comment:
You should use `array.includes(...)` here if you are okay with truthy checks.  `array.some(...)` is the better choice if you need to provide a test function like you have here.  See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
yeah, I guess using includes is correct here, I often end up using find thought as refernces are not found properly otherwise